core/pkg/hostsensorutils/hostsensorworkerpool.go

```diff
@@ -0,0 +1,91 @@
+package hostsensorutils
+
+import (
+	"sync"
+
+	"github.com/armosec/kubescape/v2/core/cautils/logger"
+	"github.com/armosec/kubescape/v2/core/cautils/logger/helpers"
+	"github.com/armosec/opa-utils/objectsenvelopes/hostsensor"
+)
+
+const noOfWorkers int = 10
```

Comment:
I will expect the number of workers to decrease when there are less than 10 nodes. Is that possible at all?